### ISSUE TITLE
Implement TMS and XYZ Baselayers

### ISSUE THREAD

**Issue by MattReimer** [Riverscapes/RaveAddIn#139]
We now have the ability to use more than just WMS baselayers.

We added this to the BaseMaps.xml file here
https://github.com/Riverscapes/RiverscapesXML/pull/85

This request came from an issue requesting Google and Bing baseLayers
https://github.com/Riverscapes/RiverscapesXML/issues/74

I'm going to handle the QRave side of things and I have no idea if this is even possible in Arc (but I'm proceeding on the assumption that it is)

**Comment by joewheaton** [comment]
@philipbaileynar you could ask @tyler1218hatch to check this out (pinging @wally-mac FYI).

**Comment by wally-mac** [comment]
Yes, please see if this is doing in ArcGIS pro @tyler1218hatch. @philipbaileynar please feel free to direct Tyler as needed.

**Comment by tyler1218hatch** [comment]
@philipbaileynar I'm a bit confused on what to do here. Should I just wait until Lauren adds the Google maps layer, then test that out in Arc once she's done?

**Comment by philipbaileynar** [comment]
@tyler1218hatch no action needed.

@MattReimer has taken care of this in QRAVE. I will implement in ArcRAVE.
